src/main/resources/site/parts/employee/employee.jsx

```diff
@@ -1,109 +1,195 @@
-import React from 'react'
-import { Button, Title, Link, LeadParagraph, Paragraph, Text } from '@statisticsnorway/ssb-component-library'
-import PropTypes from 'prop-types'
-import { Share2, Send, Smartphone, Eye, Download, ExternalLink } from 'react-feather'
+import React from 'react';
+import { Button, Title, Link, LeadParagraph, Paragraph, Text, Accordion } from '@statisticsnorway/ssb-component-library';
```

Comment:
Unused import 'Text' from '@statisticsnorway/ssb-component-library' and 'Image' from 'react-feather'.